packages/scaffold-utils/src/EthersTypesUtil.ts

```diff
@@ -28,7 +29,7 @@ export interface RequestArguments {
 
 export interface Provider {
   request: <T>(args: RequestArguments) => Promise<T>
-  on: <T>(event: string, listener: (data: T) => void) => void
+  on: <T>(event: any, listener: (data: T) => void) => void
```

Comment:
little hack I need to fix somehow, since coinbase events are hard-typed then this expects actual rpc method names instead of `string`.

will def fix before merging 🖖